libs/datasets/timeseries.py

```diff
@@ -316,6 +316,16 @@ def _timeseries_latest_values(self) -> pd.DataFrame:
         unduplicated_and_last_mask = ~long.index.duplicated(keep="last")
         return long.loc[unduplicated_and_last_mask, :].unstack()
 
+    def latest_in_static(self, field: FieldName) -> "MultiRegionDataset":
+        """Returns a new object with the latest values from timeseries 'field' copied to static."""
+        latest_series = self._timeseries_latest_values()[field]
+        if field in self.static.columns and self.static[field].notna().any():
+            # This is not expected, so detect it and don't continue
+            raise ValueError("Can only copy field when static is unset")
+        static_copy = self.static.drop(columns=[field], errors="ignore")
+        static_copy.insert(len(static_copy.columns), field, latest_series)
```

Comment:
Is this equivalent to:
```
static_copy[field] = latest_series
```
?

Is there a reason to prefer insert()?  Seems undesirable to have to specify the column position since I assume we don't care?

Reply:
Nice catch. I started by only calling `drop` when the column was present but removed a branch and didn't simplify the field creation.